cli/util.go

```diff
@@ -311,6 +311,39 @@ func jetstreamOpts() []jetstream.JetStreamOpt {
 	return res
 }
 
+// New function doesn't set a timeout. New jetstream API uses contexts for timeouts
```

Comment:
hmm, there is a default timeout that would be used all over for timeouts when no context is given, might be worth setting that just from opts.Timeout to be safe